tests/unit-tests/test-class-admin.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+class Sensei_Class_Admin_Test extends WP_UnitTestCase {
+
+	/**
+	 * Constructor function
+	 */
+	public function __construct() { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found
```

Comment:
Why do we need this constructor to exist?

Reply:
Actually I confirmed now and we really don't need =P
I had copied from the pattern of the other tests.
But I removed now. ;)